src/layers/CanvasLayer.ts

```diff
@@ -18,6 +18,7 @@ export abstract class CanvasLayer extends Layer {
       this.canvas = canvas;
       event.elm.appendChild(canvas)
     }
+    this.canvas.setAttribute('id', this.id.toString());
```

Comment:
I think eslint would complain if we don't use `${this.id}`

Reply:
i believe that rule applies to strings that are concatenated, but i'll change it nonetheless